src/types.d.ts

```diff
@@ -553,11 +553,17 @@ export interface ContiguousDataAttributes {
   hash?: string;
   dataRoot?: string;
   size: number;
+  offset: number;
   contentEncoding?: string;
   contentType?: string;
   rootTransactionId?: string;
   rootParentOffset?: number;
   dataOffset?: number;
+  itemSize?: number;
```

Comment:
this is the raw data size with headers + data, it currently conflicts with the existing `size` attribute, which is just the data size and provided as the content-length in the headers